packages/api-v4/src/resources/types.ts

```diff
@@ -0,0 +1,9 @@
+export type AccountResource = {
+  resource_type: string;
```

Comment:
Can this be more specific?

For example:
```suggestion
  resource_type: 'account' | 'firewall';
```

Reply:
sure, I've updated it